eclcli/bare/bareclient/v2/metadata.py

```diff
@@ -11,20 +11,20 @@ def __repr__(self):
 class MetadataManager(base.Manager):
     resource_class = Metadata
 
-    def get(self, server_id,key):
-        url = "/servers/%s/metadata/%s" % (server_id,key)
+    def get(self, server_id, key):
+        url = "/servers/%s/metadata/%s" % (server_id, key)
         return self._get(url, "metadata")
 
-    def delete(self, server_id,key):
-        url = "/servers/%s/metadata/%s" % (server_id,key)
+    def delete(self, server_id, key):
+        url = "/servers/%s/metadata/%s" % (server_id, key)
         return self._delete(url)
 
     def list(self, server_id):
         url = "/servers/%s/metadata" % server_id
         return self._get(url, "metadata")
 
     def update(self, server_id, key, body):
-        url = "/servers/%s/metadata/%s" % (server_id,key)
+        url = "/servers/%s/metadata/%s" % (server_id, key)
         return self._update(url, body, "metadata")
 
 # Temporary bug fix , Currently update is replacing the metadata and create is merging the data
```

Comment:
The indent of the comment is misaligned.
```suggestion
    # Temporary bug fix , Currently update is replacing the metadata and create is merging the data
```